File: exercises-python/2. Problemas de estructura condicional/13.py
'''
13.Desarrolle el programa que lea un número entero positivo de tres cifras
y determine si las cifras son o no consecutivas (en orden ascendente o descendente).
En caso contrario mostrar mensaje.
'''

import os
os.system("cls")

#Datos
num = input("Número: ")
num1 = int(num[0])
num2 = int(num[1])
num3 = int(num[2])

#Conparación
if num1 == num2 - 1 == num3 - 2: print("Ascendente")
elif num1 == num2 + 1 == num3 + 2: print("Descendente")
else: print("No son consecutivas.")

File: exercises-cpp/2. Condicionales/13.cpp
#include <iostream>
#include <cstdio>
#include <cmath>
using namespace std;

int main()
{
    system("cls");
    
    int num;
    int i = 0;
    double cifras[3];
    double num1,num2,num3;

    printf("Número "); cin>>num;

    while (num > 0){
        cifras[i]= num % 10;
        num = num / 10;
        i++;
    }
    num1 = cifras[2];
    num2 = cifras[1];
    num3 = cifras[0];

    if ((num1) - (num2 - 1) + (num3 - 2) == num1) printf("Ascendente");
    else if ((num1) - (num2 + 1) + (num3 + 2) == num1) printf("Descendente");
    else printf("No son consecutivas.");;
    }